USFMConverter/UI/Pages/OptionView.axaml.cs

```diff
@@ -19,43 +21,67 @@ private void InitializeComponent()
         {
             AvaloniaXamlLoader.Load(this);
 
-            blurredArea = this.FindControl<Border>("BlurredArea");
-            blurredArea.AddHandler(PointerPressedEvent, OnCloseClick);
-
             outputFormatCb = this.FindControl<ComboBox>("OutputFormatSelector");
+            int lastUsedFormatIndex = SettingManager.LoadMostRecentFormat()?.FormatIndex ?? 0;
+            outputFormatCb.SelectedIndex = lastUsedFormatIndex; // pickup last used format
+            ShowFormatView();
             outputFormatCb.AddHandler(ComboBox.SelectionChangedEvent, OnOuputFormatSelect);
-
-            optionView = this.FindControl<UserControl>("OptionView");
         }
 
         private void OnOuputFormatSelect(object? sender, SelectionChangedEventArgs e)
         {
-            string selectedFormat = ((ComboBoxItem)outputFormatCb.SelectedItem).Tag.ToString();
-            
+            string selectedFormat = ((ComboBoxItem) outputFormatCb.SelectedItem).Tag.ToString();
+            ShowFormatView();
+            SettingManager.SaveSettings((ViewData)DataContext);
```

Comment:
Seems a little strange here that you're saving settings and then loading them again

Reply:
maybe not good wording? It means SaveCurrentSetting(currentFormat) and LoadSettings(selectedFormat)